_episodes/01-sql-basic-queries.md

```diff
@@ -44,9 +44,7 @@ Or we can select all of the columns in a table using the wildcard *
 
 ### Limiting results
 
-Sometimes you don't want to see all the results you just want to get a sense of
-of what's being returned. In that case you can use the LIMIT command. In particular
-you would want to do this if you were working with large databases.
+Sometimes you don't want to see all the results you just want to get a sense of what's being returned. In that case, you can use the LIMIT command. In particular, you would want to do this if you were working with large databases.
```

Comment:
"want to see all the results you just" -> "want to see all the results, you just"

Reply:
"you can use the LIMIT command" -> "you can use the `LIMIT` command"